src/components/Buttons/ButtonWithQrCode/ButtonWithQrCode.vue

```diff
@@ -4,7 +4,7 @@
       <div class="the-button large-round-button-green-filled">
         {{ buttonname }}
       </div>
-      <div class="qrcode-container">
+      <div v-if="false" class="qrcode-container">
```

Comment:
Just delete this since setting v-if to false makes sure that this snippet wont be rendered